src/networkAddresses.ts

```diff
@@ -180,6 +180,7 @@ export const NETWORK_ADDRESSES = {
     CHAINLINK_ORACLE: "0x4FC29E1d3fFFbDfbf822F09d20A5BE97e59F66E5",
   },
   opsepolia: {
+    VTREASURY: "0x5A1a12F47FA7007C9e23cf5e025F3f5d3aC7d755",
     NORMAL_TIMELOCK: "0xd57365EE4E850e881229e2F8Aa405822f289e78d", // opsepolia Multisig
```

Comment:
I would keep NORMAL_TIMELOCK key empty unitl we have multichain governance

Reply:
Removed, 8a144ff308eb2a83393043ca965a180418f91bda